stable/prometheus-to-sd/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.2.2
+appVersion: 0.2.3
```

Comment:
You're bumping `appVersion` from `0.2.2` to `0.2.3` instead of increase `version` from `0.1.0` to `0.1.1`